README.adoc

```diff
@@ -276,17 +281,17 @@ query/server.xml
 include::finish/query/src/main/liberty/config/server.xml[]
 ----
 
-Add the [hotspot=grpc file=3]`grpc` and [hotspot=grpcClient file=3]`grpcClient` features to the Liberty server configuration file. Both features enable gRPC server and client compatibility with the Liberty server. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` feature with the `headersToPropagate` and `target` attributes. If you want to learn more about `grpcClient` feature configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
+Add the [hotspot=grpc file=3]`grpc` and [hotspot=grpcClient file=3]`grpcClient` features to the Liberty query server configuration file. Both features enable gRPC server and client compatibility with the Liberty server. Configure the [hotspot=grpcClientConfig file=3]`grpcClient` feature with the `headersToPropagate` and `target` attributes. If you want to learn more about `grpcClient` feature configuration, see the https://openliberty.io/docs/latest/reference/config/grpcClient.html[GRPC Client Properties^].
 
 Because you are running the `system` and `query` services in dev mode, the changes that you made were automatically picked up. You’re now ready to check out your application in your browser.
 
 // static guide instruction
 ifndef::cloud-hosted[]
-Point your browser to the http://localhost:9081/query/properties/os.name URL to test out the unary service. You should see your operating system name. You can try out the URL with other property name.
+Point your browser to the http://localhost:9081/query/properties/os.name URL to test out the unary service. Your operating system name is displayed. You can try out the URL with other property name.
 endif::[]
 // cloud hosted instruction
 ifdef::cloud-hosted[]
-Click the following button to visit the ***/query/properties/os.name*** endpoint to test out the unary service. You should see your operating system name. You can try out the URL with other property name.
+Click the following button to visit the ***/query/properties/os.name*** endpoint to test out the unary service. Your operating system name is displayed. You can also try out the URL with other property names.
```

Comment:
```suggestion
Click the following button to visit the ***/query/properties/os.name*** endpoint to test out the unary service. Your operating system name is displayed. You can also try out the URL with other property names but they will return an exception because the unary RPC only returns a single property.
```

Reply:
After discussion, we think it is better to leave the last sentence out to clear up confusion ("properties" is being used in two different contexts within the guide- it means os properties but also system properties).

Addressed in https://github.com/OpenLiberty/draft-guide-grpc-intro/pull/12/commits/69cf4819db94af6e8c3e92b65e5084f2be711e15.